src/init.c

```diff
@@ -289,9 +289,18 @@ void segv_handler(int sig, siginfo_t *info, void *context)
         sigprocmask(SIG_UNBLOCK, &sset, NULL);
         jl_throw(jl_memory_exception);
     }
+#ifdef SEGV_EXCEPTION
+    else {
```

Comment:
Maybe, but I'd really like to catch everything, whether it's SIGSEGV or SIGBUS (not sure what else gets trapped here). I see that the julia side error name is then misleading, but I think that's fine.